packages/ui/src/theme/css/component/stepperField.scss

```diff
@@ -42,31 +42,29 @@
   );
 }
 
-.amplify-stepperfield__button--decrease {
+.amplify-stepperfield__button--decrease,
+.amplify-stepperfield__button--increase {
```

Comment:
Consolidated common styles for increase/decrease buttons, such as `background-color`, `disabled` and `quiet` to reduce duplication.